app.py

```diff
@@ -420,33 +420,135 @@ def create_course(course_ok_id, ok_id=None):
+                            video_index = video_index,
+                            link = link, 
+                            db = db
+                        )
+                        return jsonify(success=True), 200
+                    except ValueError as e:
+                        return jsonify(success=False, message=str(e)), 500
+            return jsonify(success=False, message="Only instructors can create resources"), 403
+    return jsonify(success=False, message="Can only create a resource on Hermes for an OK course you are a part of"), 403
+
+@app.route('/course/<course_ok_id>/lecture/<lecture_url_name>/video/<int:video_index>/delete_vitamin/<int:vitamin_index>', methods=["DELETE"])
+@validate_and_pass_on_ok_id
+def delete_vitamin(course_ok_id, lecture_url_name, video_index, vitamin_index, ok_id=None):
+    """Deletes a specified vitamin within a specific video of a lecture in a given course."""
+    user_courses = get_updated_user_courses()
+    int_course_ok_id = int(course_ok_id)
+    for course in user_courses:
+        if course['course_id'] == int_course_ok_id:
+            if course['role'] != consts.INSTRUCTOR:
+                return jsonify(success=False, message="Only instructors can delete vitamins"), 403
+            db[Vitamin.collection].delete_one(
+                {
+                    'course_ok_id': course_ok_id,
+                    'lecture_url_name': lecture_url_name,
+                    'video_index': video_index,
+                    'vitamin_index': vitamin_index
+                }
+            )
+            return jsonify(success=True), 200
+    return jsonify(success=False, message="Can only delete a vitamin on Hermes for an OK course you are a part of"), 403
+
+@app.route('/course/<course_ok_id>/lecture/<lecture_url_name>/video/<int:video_index>/delete_resource/<int:resource_index>', methods=["DELETE"])
+@validate_and_pass_on_ok_id
+def delete_resource(course_ok_id, lecture_url_name, video_index, resource_index, ok_id=None):
+    """Deletes a specified resource within a specific video of a lecture in a given course."""
+    user_courses = get_updated_user_courses()
+    int_course_ok_id = int(course_ok_id)
+    for course in user_courses:
+        if course['course_id'] == int_course_ok_id:
+            if course['role'] != consts.INSTRUCTOR:
+                return jsonify(success=False, message="Only instructors can delete resources"), 403
+            db[Resource.collection].delete_one(
+                {
+                    'course_ok_id': course_ok_id,
+                    'lecture_url_name': lecture_url_name,
+                    'video_index': video_index,
+                    'resource_index': resource_index
+                }
+            )
+            return jsonify(success=True), 200
+    return jsonify(success=False, message="Can only delete a resource on Hermes for an OK course you are a part of"), 403
+
+# @app.route('/course/<course_ok_id>/create_piazza_bot', methods=["POST"])
```

Comment:
Probably just want to delete this to avoid confusion when we merge

Reply:
Thanks, good catch. Removed now